<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaMenu</class>
 <widget class="QMainWindow" name="TelaMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableDados">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>641</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBarraPesquisa">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Pesquise o aluno aqui!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPesquisar">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;h1&gt;Bem vindo ao nosso sistema de Cadastro de Alunos!
&lt;/h1&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lApresentar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;h3&gt;...&lt;/h3&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>130</y>
      <width>121</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnAdicionar">
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditar">
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExcluir">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Log out</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
